charts/seed-monitoring/charts/grafana/templates/grafana-dashboards-configmap.yaml

```
@@ -14,8 +14,15 @@ data:
 {{ end }}
 {{ else }}
 {{ range $name, $bytes := .Files.Glob "dashboards/**.json" }}
+  {{ if eq $name "dashboards/operators/vpn-dashboard.json" }}
+  {{ if not $.Values.konnectivityTunnel.enabled }}
   {{ base $name }}: |-
 {{ toString $bytes | indent 4}}
+  {{ end }}
+  {{ else }}
+  {{ base $name }}: |-
+{{ toString $bytes | indent 4}}
+  {{ end }}
```

Can we simplify this?
```suggestion
  {{ if not and eq $name "dashboards/operators/vpn-dashboard.json" $.Values.konnectivityTunnel.enabled }}
  {{ base $name }}: |-
{{ toString $bytes | indent 4}}
  {{ end }}
```

---

you mean shorten? I think it is more clear to read with two if conditions.

---

As you like, no strong opinion here :)